dna/course/zomes/courses/code/src/section/anchor.rs

```diff
@@ -0,0 +1,62 @@
+#[derive(Serialize, Deserialize, Debug,self::DefaultJson, Clone)]
+pub struct SectionAnchor {
+    pub title: String, 
+    pub timestamp: u64,
+}
+
+impl AnchorTrait for SectionAnchor {
+    fn entry_type() -> String {
+        String::from("section_anchor")
+    }
+    fn link_to() -> String {
+        Section::entry_type()
+    }
+    fn link_type() -> String {
+        "section_anchor->section".to_owned()
+    }
+}
+
+impl SectionAnchor {
+    pub fn new(title: String, timestamp: u64) -> Self {
+        SectionAnchor {
+            title:title, 
+            timestamp: timestamp,
+        }
+    }
+}
+
+pub fn section_anchor_def() -> ValidatingEntryType {
+    entry!(
+        name: SectionAnchor::entry_type(),
+        sharing: Sharing::Public, 
+        validation_package: || {
+            hdk::ValidationPackageDefinition::Entry
+        },
+        validation: | validation_data: hdk::EntryValidationData<SectionAnchor>| {
+            match validation_data{
+                EntryValidationData::Create { .. } => {
+                    Ok(())
+                }, 
+                EntryValidationData::Modify { .. } => {
+                    Ok(())
+                }, 
+                EntryValidationData::Delete { .. } => {
+                    Ok(())
+                }
+            }
+        }, 
+        links:[
```

Comment:
we'll need another link, from section anchor to content entry, for content to be discoverable. Tip: link_type for this link can be defined as const variable to be re-used in section::handlers later